website/content/userguide/install/tailscale.md

```diff
@@ -19,58 +19,88 @@ remotely (no static DHCP leases, port-forwarding and DynDNS required!), or even
 to secure your communication when gokrazy is [connected to an unencrypted WiFi
 network](/userguide/unencrypted-wifi/).
 
+{{% notice note %}}
+Tailscale currently uses [Userspace networking] mode on gokrazy, because
+for `tun` mode, Tailscale currently requires components that gokrazy does not
+provide. For accessing the services on your gokrazy installation, the Userspace
+networking mode works fine, though 🥳 .
+
+[Userspace networking]: https://tailscale.com/kb/1112/userspace-networking/ "Userspace networking mode (for containers)"
+{{% /notice %}}
+
+
+## Requirements
+ * Package `tailscale.com` v1.22.1 or later (latest version used automatically unless you have the package already in go.mod)
+ * Volume `/perm/` needs to be initialized (instructions use `github.com/gokrazy/mkfs` to initialize)
+to persist authentication over reboots.
+
+
 ## Step 1. set command-line flags
 
-We need to specify the following flags for the `tailscaled` daemon (see [Package
+We need to specify the following flags for the `tailscale` to bring up
+connection (see [Package
 config: flags and environment variables](/userguide/package-config) if you’re
 unfamiliar with this mechanism):
 
+**Option A: interactive authentication**
+
 ```shell
-mkdir -p flags/tailscale.com/cmd/tailscaled
-echo '--statedir=/perm/tailscaled/' > flags/tailscale.com/cmd/tailscaled/flags.txt
-echo '--tun=userspace-networking' >> flags/tailscale.com/cmd/tailscaled/flags.txt
+mkdir -p flags/tailscale.com/cmd/tailscale
+cat > flags/tailscale.com/cmd/tailscale/flags.txt <<EOF
+up
+EOF
 ```
 
-`tailscaled` requires the `--statedir` flag, so we need to set it
-explicitly. `/perm/tailscaled` is the working directory of the `tailscaled`
-process and will contain the `tailscaled.sock` socket, so it makes sense to
-place the state file into the same directory.
+**Option B: unattended authentication with auth key**
 
-The `--tun=userspace-networking` flag selects the [Userspace
-networking](https://tailscale.com/kb/1112/userspace-networking/) mode.
+Navigate to [Tailscale console] and open Settings / Keys. Generate auth key.
```

Comment:
The `[Tailscale console]` link seems broken when previewing the website locally.

Reply:
A-ha. It worked when I wrote it but broke when I moved the other section with same link to notice box. Apparently that scope is somehow different. Good catch.